packages/strapi-plugin-i18n/admin/src/components/LocaleRow/index.js

```diff
@@ -8,15 +8,35 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { getTrad } from '../../utils';
 
-// Fake permissions
-const canUpdate = true;
-const canDelete = true;
-
-const LocaleSettingsPage = ({ locale, onDelete }) => {
+const LocaleSettingsPage = ({ locale, onDelete, onEdit }) => {
```

Comment:
Sorry I was not clear about the permissions. We need to keep the canUpdate and canDelete but use them as default props, Your component should be as follows:

```
const LocaleSettingsPage = ({ canDelete, canUpdate, locale, onDelete, }) => {}

LocaleSettingsPage.defaultProps = {
canDelete: true,
canUpdate: true
};

LocaleSettingsPage.propTypes = {
canDelete: PropTypes.bool,
canUpdate: PropTypes.bool
};

```